pyproject.toml

```diff
@@ -3,7 +3,8 @@ name = "run-detection"
 readme = "README.md"
 version = "0.0.1"
 dependencies = [
-    "stomp.py==8.1.0"
+    "stomp.py==8.1.0",
+    "confluent-kafka==1.9.2"
```

Comment:
Does the confluent kafka still work fine on none-confluent based kafka systems?

Reply:
Yes. It's a python wrapper of [librdkafka](https://github.com/confluentinc/librdkafka), maintained and actively developed by confluent.